crates/kicad-parser/src/writer.rs

```diff
@@ -0,0 +1,146 @@
+use sexp::{atom_f, atom_s, Sexp};
+
+use crate::{
+    board::{BoardLayer, Footprint, KicadBoard},
+    graphics::{GraphicArc, GraphicCircle, GraphicLine, GraphicRect},
+};
+
+pub fn write_board<W: std::io::Write>(
+    writer: &mut W,
+    board: &KicadBoard,
```

Comment:
Should this instead be `KicadPcb`? I think my original plan was for `KicadBoard` to hold the actual board data like copper, footprints, silkscreens, etc, while `KicadPcb` would hold a `KicadBoard`, as well as extra data like the version and other metadata.